libsql/src/local/database.rs

```diff
@@ -90,6 +90,14 @@ impl Database {
 
         let endpoint = coerce_url_scheme(endpoint);
         let remote = crate::replication::client::Client::new(
+            connector.clone(),
+            endpoint.as_str().try_into().unwrap(),
+            auth_token.clone(),
+            version.as_deref(),
+            http_request_callback.clone(),
+        )
+        .unwrap();
```

Comment:
You will probably need to merge to get these fixes as well for the unwraps https://github.com/tursodatabase/libsql/commit/d24435e564db895dc9578eada7b8bbcabdc88df5

Reply:
The change is based on main. It seems that you missed this place. I will add a commit that fixes it.

Reply:
Added a commit that removes the unwraps